ingestion_server/test/integration_test.py

```diff
@@ -398,29 +444,28 @@ def tearDownClass(cls) -> None:
             if conn:
                 conn.close()
 
-    @pytest.mark.order(1)
     def test_list_tasks_empty(self):
         res = requests.get(f"{ingestion_server}/task")
         res_json = res.json()
         msg = "There should be no tasks in the task list"
         self.assertEqual(res_json, [], msg)
 
-    @pytest.mark.order(2)
+    @pytest.mark.order(after="test_list_tasks_empty")
```

Comment:
Switching to `after` instead of a numbered order allows inserting new tests in the future without having to update all the numbers for each subsequent test (or adding the tests at the end, to avoid that). See https://pytest-dev.github.io/pytest-order/stable/usage.html#order-relative-to-other-tests for documentation.

Reply:
Such a good idea!!